docs/UsersGuide/gds/seqgen.md

```diff
@@ -0,0 +1,58 @@
+# Sequencing In F´
+
+F´ supports a very basic sequence format that allows for executing F´ commands at absolute times and at times relative
+to previous commands. The sequence uses the F´ dictionary to translate the human readable sequence file into a minimized
+binary format for upload and running using the command sequencer.
+
+Compiling this sequence is done with the `fprime-seqgen` utility and can be run once uploaded by issuing the `*.CS_RUN`
+command of any command sequencer instance defined in the flight system.
+
+## Writing an F´ Sequence File
+
+F´ sequence files consist of lists of commands. These commands start with a time argument, followed by the command
+mnemonic, and lastly any arguments to command.  See the example below, which are all pulled from the example sequence
+file found here: [simple_sequence.seq](https://github.com/nasa/fprime/blob/devel/Gds/examples/simple_sequence.seq).
+
+```
+A2015-075T22:32:40.123 cmdDisp.CMD_NO_OP
```

Comment:
Could you possibly clarify the time format of the relative and absolute date strings?

Reply:
Fixed.